app/src/ui/publish-repository/publish-repository.tsx

```diff
@@ -65,6 +66,7 @@ export class PublishRepository extends React.Component<
   private async fetchOrgs(account: Account) {
     const api = API.fromAccount(account)
     const orgs = await api.fetchOrgs()
+    orgs.sort((a, b) => compare(a.login, b.login))
```

Comment:
`ReadOnlyArray` doesn't have `sort` defined. Probably need to do something like this:

```ts
    const orgs = await api.fetchOrgs()
    const sortedArray = new Array<IAPIUser>(...orgs)
    sortedArray.sort((a, b) => compare(a.login, b.login))
    this.setState({ orgs: sortedArray })
```